ci/docker/install/ubuntu_arm_qemu_bin.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# build and install are separated so changes to build don't invalidate
+# the whole docker cache for the image
+
+set -exuo pipefail
+
+REMOTE="https://s3-eu-west-1.amazonaws.com/mxnet-edge-public/qemu"
```

Comment:
Where does this come from and how is it generated? How's the licensing for the files hosted in there? If there's no other reliable source, it should be moved to the mxnet-ci S3 bucket

Reply:
Yes, please have a look at the bucket "mxnet-ci-prod-slave-data" in the mxnet-ci-prod account

Reply:
Why is one bucket better than the other? I will have to set up writing for that bucket, which is a PITA.

Reply:
Because the bucket I mentioned is owned by the mxnet-ci-prod account while the other one is owned by the mxnet-edge account. Simply a matter of separating concerns and avoiding accidental deletes. For the CI prod account, we know that these things shouldn't be touched while for the edge account we should be free to move stuff without impacting the productive CI